card/mod/standard/set/type/session.rb

```diff
@@ -23,7 +23,7 @@ def content
 end
 
 def content= val
-  Env.session[key] = val
+  self.db_content = Env.session[key] = val
```

Comment:
Fixes session-related source issue in wikirate cukes.  Bug appears to have been introduced here: https://github.com/wikirate/wikirate/compare/43795ab...725bc17

I'm not sure why this became necessary, but the most likely candidate so far is  `attribute_is_changing?`

Without this, db_content is often nil, so store_in_session just sticks a nil value in the session.